src/main/java/com/feywild/feywild/world/gen/OreType.java

```diff
@@ -7,64 +7,85 @@
-                    CountOnEveryLayerPlacement.of(ConstantInt.of(getSpawnWeight()))
-            ));
+
+        this.stoneFeature = new LazyValue<>(() -> {
+            OreConfiguration oreFeatureConfig = new OreConfiguration(OreFeatures.STONE_ORE_REPLACEABLES, stoneBlock.defaultBlockState(), getMaxVeinSize());
+
+            return Registry.register(BuiltinRegistries.PLACED_FEATURE, Objects.requireNonNull(stoneBlock.getRegistryName()),
+                    Feature.ORE.configured(oreFeatureConfig).placed(OrePlacement.commonOrePlacement(getSpawnWeight(),
+                            HeightRangePlacement.triangle(VerticalAnchor.absolute(getMinHeight()), VerticalAnchor.absolute(getMaxHeight())))
+                    ));
+        });
+
+        this.deepSlateFeature = new LazyValue<>(() -> {
+            OreConfiguration oreFeatureConfig = new OreConfiguration(OreFeatures.DEEPSLATE_ORE_REPLACEABLES, deepSlateBlock.defaultBlockState(), getMaxVeinSize());
+
+            return Registry.register(BuiltinRegistries.PLACED_FEATURE, Objects.requireNonNull(deepSlateBlock.getRegistryName()),
+                    Feature.ORE.configured(oreFeatureConfig).placed(OrePlacement.commonOrePlacement(getSpawnWeight(),
+                            HeightRangePlacement.triangle(VerticalAnchor.absolute(getMinHeight()), VerticalAnchor.absolute(getMaxHeight())))
+                    ));
         });
+
         this.alfheimFeature = new LazyValue<>(() -> {
             OreConfiguration oreFeatureConfig = new OreConfiguration(FeywildOreGen.ALFHEIM_STONE, alfheimBlock.defaultBlockState(), getMaxVeinSize());
-            return Registry.register(BuiltinRegistries.PLACED_FEATURE, Objects.requireNonNull(block.getRegistryName()), Feature.ORE.configured(oreFeatureConfig).placed(
-                    HeightRangePlacement.triangle(VerticalAnchor.absolute(getMinHeight()), VerticalAnchor.absolute(getMaxHeight())),
-                    CountOnEveryLayerPlacement.of(ConstantInt.of(getSpawnWeight()))
-            ));
+            return Registry.register(BuiltinRegistries.PLACED_FEATURE, Objects.requireNonNull(stoneBlock.getRegistryName()),
+                    Feature.ORE.configured(oreFeatureConfig).placed(OrePlacement.commonOrePlacement(getSpawnWeight(),
+                            HeightRangePlacement.triangle(VerticalAnchor.absolute(getMinHeight()), VerticalAnchor.absolute(getMaxHeight())))
+                    ));
         });
     }
 
     public static OreType get(Block block) {
         for (OreType ore : values()) {
-            if (block == ore.block) {
+            if (block == ore.stoneBlock) {
                 return ore;
             }
         }
         return null;
     }
 
-    public Block getBlock() {
-        return this.block;
+    public static void setupOres() {
+        for (OreType ore : values()) {
+            // Will trigger registration
+            ore.getStoneFeature();
```

Comment:
This needs to call `getDeepslateFeature()` as well. And if MythicBotany is installed, it should also call `getAlfheimFeature()`. Altough MythicBotany is not yet available for 1.18, you can still add that now, so it is there later.